src/components/admin/hotels/save-hotel/index.js

```diff
@@ -0,0 +1,228 @@
+    website: yup
+        .string('Enter website'),
+});
+
+const SaveHotelForm = ({ isOpen, handleSaveHotelModalToggle, hotelModel }) => {
+    const isEditForm = Boolean(hotelModel && hotelModel.id);
+    const initialValues = hotelModel || {
+        name: '',
+        stars: 0,
+        address: '',
+        contactName: '',
+        email: '',
+        phone: '',
+        website: '',
+        attachments: [],
+    };
+    const formik = useFormik({
+        initialValues: initialValues,
+        validationSchema: validationSchema,
+        onSubmit: async (model, formikHelper) => {
+            const newHotel = isEditForm
+                ? await HotelClient.editHotel(model)
+                : await HotelClient.addHotel(model);
+
+            if (newHotel) {
+                formikHelper.resetForm();
+            }
+        },
+        onReset: () => {
+            handleSaveHotelModalToggle();
+        }
+    });
+
+    function handleFileAdd(file) {
+        formik.setFieldValue('attachments', [...formik.values.attachments, { name: file.name, blob: file }]);
+    }
+
+    function handleAttachmentRemove(attachment) {
+        const attachments = formik.values.attachments;
+        const index = formik.values.attachments.indexOf(attachment);
+        if (index > -1) {
+            attachments.splice(index, 1);
+            formik.setFieldValue('attachments', attachments);
+        }
+    }
+
+    const inputMb = 1;
+    const dialogTitle = isEditForm ? 'Edit new hotel' : 'Add new hotel';
+
+    return (
+        <Dialog open={isOpen} maxWidth="lg">
+            <DialogTitle><HotelIcon />{dialogTitle}</DialogTitle>
+            <DialogContent>
+                <Container>
+                    <form onSubmit={formik.handleSubmit} noValidate autoComplete="off">
+                        <Grid container direction="row" justify="center" alignItems="flex-start" spacing={3}>
+                            <Grid item xs={6}>
+                                <Box mb={inputMb}>
+                                    <TextField
+                                        fullWidth
+                                        id="name"
+                                        name="name"
+                                        placeholder="Name"
+                                        value={formik.values.name}
+                                        onChange={formik.handleChange}
+                                        error={formik.touched.name && Boolean(formik.errors.name)}
+                                        helperText={formik.touched.name && formik.errors.name}
+                                        variant="outlined"
+                                        size="small"
+                                    />
+                                </Box>
+                                <FormControl className="rating-form-control" fullWidth>
+                                    <Grid container direction="row" justify="center" alignItems="flex-start">
```

Comment:
`direction="row" justify="center" alignItems="flex-start"` are redundant for this case.

Reply:
Fixed.